### PR TITLE
:tada: Add terms check to register

### PR DESCRIPTION
**Pull request by hirunatan**
> Please provide enough information so that others can review your pull request:

> Explain the **details** for making this change. What existing problem does the pull request solve?

> Screenshots/GIFs

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,2665 @@
+{
+  "requires": true,
+  "lockfileVersion": 1,
```
Comment (niwinz) [inline package-lock.json]: This file should not be in the repository.

Comment (hirunatan) [reply package-lock.json]: Hehehe... it keeps sneaking in each time whe get distracted. I'll remove it.

**Comment by niwinz** [comment]
This should point `main` branch

**Comment by niwinz** [comment]
merged

**Pull request closed without merging**